Source/Localization/es-UY.lproj/Generated.strings

```diff
@@ -53,8 +53,8 @@
 "Text.name" = "Nombre";
 "Text.bio" = "Bio";
 "Text.seeMore" = "Ver más";
-"Text.likesThis" = "Le gusta";
-"Text.dislikesThis" = "No le gusta";
+"Text.likesThis" = "Likes this";
+"Text.dislikesThis" = "Dislikes this";
```

Comment:
like here

Reply:
Why?

Reply:
maybe just here actually

Reply:
Crowdin seems to think there was never any translation for this string at all. https://crowdin.com/project/planetary/activity-stream?lang=75&translation_id=66384.

Tracing this back, it looks like these strings were translated manually in https://github.com/planetary-social/planetary-ios/commit/f989615f4 and Crowdin didn't pick them up as translated. Is it supposed to? 🤷

Reply:
I have fixed these in Crowdin and I'm going to go ahead and merge this. I guess we'll need to avoid translating things directly in the source files until we get this figured out.